### DISCUSSION TITLE
Can I create a theme for Blink Shell?

### DISCUSSION THREAD

**Discussion by carloscabanero** [blinksh/blink#1527] [Q&A] [answered]
How can I change the cursor color in my terminal? Can I change the cursor shape? Can I add a glow effect to my fonts?

**Comment by carloscabanero** [comment, marked as answer]
Making a custom theme for Blink Shell is simple. Let's start by creating a Gist named `mytheme.js`. Now paste the below content to it, and save it.

```javascript
black = "#1C1C1C";
red = "#AF5F5F"; // red
green = "#5F875F"; // green
yellow = "#87875F"; // yellow
blue = "#5F87AF"; // blue
magenta = "#5F5F87"; // pink
cyan = "#5F8787"; // cyan
white = "#6C6C6C"; // light gray
lightBlack = "#444444"; // medium gray
lightRed = "#FF8700"; // red
lightGreen = "#87AF87"; // green
lightYellow = "#FFFFAF"; // yellow
lightBlue = "#8FAFD7"; // blue
lightMagenta = "#8787AF"; // pink
lightCyan = "#5FAFAF"; // cyan
lightWhite = "#FFFFFF"; // white

t.prefs_.set("color-palette-overrides", [
  black,
  red,
  green,
  yellow,
  blue,
  magenta,
  cyan,
  white,
  lightBlack,
  lightRed,
  lightGreen,
  lightYellow,
  lightBlue,
  lightMagenta,
  lightCyan,
  lightWhite,
]);

t.prefs_.set("foreground-color", "#bbbbbb");
t.prefs_.set("background-color", "#121212");
t.prefs_.set("cursor-color", "rgba(187,187,187,0.5)");
```

For some special effects like glow, check out: https://github.com/blinksh/blink/issues/1478.

Paste the link to the Gist at:
Go to **Settings** > **Appearance** > **Add a new theme**. Add a new name and click import.

![IMG_2381.png](https://res.craft.do/user/full/0ef94637-30d5-80f2-9fa1-0089b763dcf6/248A9189-CA8A-47BB-9C8E-DA2A09E22381_2/yneNBSVidSUwATUnFxI9NtIrUeXZKML1vmVOmZqyYhkz/IMG_2381.png)

From now on, you can choose it from the Appearance settings.

![IMG_2382.png](https://res.craft.do/user/full/0ef94637-30d5-80f2-9fa1-0089b763dcf6/FC0A273B-D03B-471E-A122-70E6461F0C20_2/3DgFNsyl6osxTypiyA28ZmUSV0Jgna3xsYm6pyEmphEz/IMG_2382.png)

#### Send us PR if you want to share your awesome theme with the community.

Or just demonstrate it in Show & Tell section on GitHub and share a direct link for others to download.

### More information

- Creating fonts & themes: [https://docs.blink.sh/advanced/creating-fonts-and-themes](https://docs.blink.sh/advanced/creating-fonts-and-themes)

**Comment by pcho** [comment]
Users interested in creation of dynamic themes adapting to system appearance can check this out: https://github.com/ng-hai/rose-pine-blinksh/tree/main/dynamic
